configator/main.py

```diff
@@ -0,0 +1,44 @@
+"""This file will use all of our functions and run the program"""
+
+import typer
+
+# Import from configator
+from configator import create_actions_config
+from configator import create_gatorgradle_yml
+from configator import generate_build_gradle
+
+#pylint: disable=too-many-arguments
+app = typer.Typer()
+@app.command()
+def main(
+
+    name: str = typer.Option("configator-generated"),
```

Comment:
It'd be cool if the default for the name of the assignment (this option) was the name of the current directory -- you can get that with `Path.cwd()`, or many other functions available in Python by default.